moscar.go

```diff
@@ -1,124 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"github.com/micromdm/squirrel/munki/datastore"
```

Comment:
imports which do not come from the standard library should be at the bottom. In your editor, if you enable `goimports` on save, import statements will be automatically added/removed as you're editing code. 

`goimports` is a drop in replacement for `gofmt` and will do both tasks.